homeassistant/components/netatmo/strings.json

```
@@ -98,6 +100,20 @@
         }
       }
     },
+    "sync_schedule": {
+      "name": "Sync heating schedule",
+      "description": "Syncs the heating schedule for Netatmo climate device.",
+      "fields": {
+        "home_id": {
+          "name": "[%key:component::netatmo::device_automation::trigger_subtype::home_id%]",
+          "description": "Home Id."
+        },
+        "schedule_id": {
+          "name": "[%key:component::netatmo::device_automation::trigger_subtype::schedule_id%]",
+          "description": "Schedule Id."
```

Same here. Add some explanation where to find that ID (just a suggestion, change as needed):

```suggestion
          "description": "The ID of a schedule defined in the Netatmo mobile app."
```